target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -19,7 +19,8 @@ case "$FIRMWARE" in
 	;;
 "ath10k/cal-pci-0000:01:00.0.bin")
 	case $board in
-	mikrotik,routerboard-922uags-5hpacd)
+	mikrotik,routerboard-922uags-5hpacd|\
+	mikrotik,routerboard-962uigs-5hact2hnt)
 		caldata_sysfsload_from_file $wlan_data 0x5000 0x844
```

Comment:
Regarding needing both pcie0 & pcie1 enabled: I _think_ when DTS SoC pcie0 is disabled, SoC pcie1 becomes linux pcie0, so this 962 board case needs to be in a different FIRMWARE case?

Reply:
You're exactly right. PR updated.